underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -340,28 +342,42 @@ protected ObjectListingChunk getObjectListingChunk(String key, boolean recursive
     key = PathUtils.normalizePath(key, PATH_SEPARATOR);
     // In case key is root (empty string) do not normalize prefix
     key = key.equals(PATH_SEPARATOR) ? "" : key;
-    ListObjectsV2Request request =
-        new ListObjectsV2Request().withBucketName(mBucketName).withPrefix(key)
-            .withDelimiter(delimiter).withMaxKeys(getListingChunkLength());
-    ListObjectsV2Result result = getObjectListingChunk(request);
-    if (result != null) {
-      return new S3AObjectListingChunk(request, result);
+    if (mConf.containsKey(PropertyKey.UNDERFS_S3A_LIST_OBJECTS_VERSION_1) && mConf
+        .getValue(PropertyKey.UNDERFS_S3A_LIST_OBJECTS_VERSION_1).equals(Boolean.toString(true))) {
```

Comment:
Unrelated to this PR, but it is unfortunate we can only get `String` back from the `UnderFileSystemConfiguration`.